src/app/interactions/interactions-results/interactions-filters/interactions-filters.component.html

```diff
@@ -4,284 +4,313 @@
             <i class="icon icon-functional icon-spacer icon-filter"></i> Filters
         </div>
         <button class="button lighter" type="button" data-toggle="interactor-species-dropdown"
-                [ngClass]="{'underline': filters.isFiltering(filterTypes.SPECIES)}">
+                [ngClass]="{'underline': filters.isFiltering(filterTypes.SPECIES) || filters.isFiltering(filterTypes.INTRA_SPECIES)}">
```

Comment:
Nice catch. I've been working on the filters for a week and haven't even noticed this.